### PR TITLE
downgrade bootstrap to v4

### PR DESCRIPTION
**Pull request by Nexucis**
I'm downgrading bootstrap to the previous version.

To be able to upgrade this version, we will have to:
* Replace `@forevolve/bootstrap-dark` because it doesn't support bootstrap 5 currently. I asked [here](https://github.com/ForEvolve/bootstrap-dark/issues/52) if they would like to support.
* Upgrade `reactstrap` to v9 and adapt the code accordingly to the breaking changes introduced.

Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
Thanks! :+1:  Important to note for others that we've been on BS 4 until dependabot decided it was time for an upgrade to 5 a few days ago: https://github.com/prometheus/prometheus/pull/9778

**Comment by roidelapluie** [comment]
I did test https://deploy-preview-9778--prometheus-react.netlify.app/alerts and it was working as expected. What was I missing?

**Comment by Nexucis** [comment]
Weeeell, here was the situation:

In `react-app/package.json` you had the dependency `bootstrap: v5`. So you could assume this version was actually installed.
But in fact, if you take a look at the dependency `@forevolve/bootstrap-dark`, it depends on `bootstrap: v4`. So in reality, it wasn't the v5 installed but the v4. 

And "fortunately" for us it was like that, because then you realized that `reactstrap` has also a peer dependency to `bootstrap v4`.

So to summarize, dependabot should never have upgraded the `bootstrap` dependency due to the constraints described above.

**Comment by roidelapluie** [comment]
Which is why it does not update package.lock.....
